CHANGELOG.md

```diff
@@ -636,6 +636,7 @@
 - Include 'source fund' and 'delivery partner short name' in CSV export
 - Prevent report mailer from sending emails to inactive users
 - Merge Academies Collective Fund and Resilient Futures GCRF strategic areas into one
+- Total budget should only total budgets from a specfic activity
```

Comment:
If you want to fix the typo (`specfic`) while fixing the conflict, the pedantic part of my brain would be very grateful. ;)